coalib/testing/LocalBearTestHelper.py

```diff
@@ -12,16 +13,38 @@
 
 
 @contextmanager
-def execute_bear(bear, *args, **kwargs):
+def execute_bear(bear, *args, **kwargs):  # pragma: no cover
```

Comment:
This patch involves printing the `stderr` and `stdout` from linter bears when there is some problem or an exception is raised. To test this feature, I will deliberately have to raise an exception and thus make the tests fail so as to see if we're printing the `stderr` and `stdout` or not and IIRC once, we raise an exception, the tests hault there itself and we wouldn't be able to proceed further. However, I've [tested this manually](https://github.com/coala/coala-bears/issues/455#issuecomment-317270508). :smile: